docs/sources/developers/plugins/migration-guide/v9.x-v10.x/_index.md

```diff
@@ -34,3 +34,54 @@ Most code targeting 9.x will continue to work without any issues. An exception i
 When writing plugins that should run on 9.x, continue to use the Vector interfaces. In this case, when targeting versions 10+, you can now use simple arrays rather than wrapper classes.
 
 To make this transition seamless, we employed the Original JavaScript Sin™. That is, we[extended the native Array prototype](https://github.com/grafana/grafana/blob/v10.0.x/packages/grafana-data/src/types/vector.ts) with several Vector methods. We will atone and undo this in v11, when Vector interfaces and classes are removed.
+
+## Update to React Router v6
+
+Starting from Grafana 10 plugins can start updating to use the v6 version of `react-router`. Overall, `react-router` v6 aims to simplify route configuration and provide a more flexible and intuitive API for developers.
```

Comment:
Q. plugins `can` or `should` migrate? the line after implies is required.

Reply:
Yeah, I think it's good question. I would say it's rather a _should_, but I guess we would need to discuss it in the team.